src/site/paragraphs/collapsible_panel.drupal.liquid

```diff
@@ -1,14 +1,47 @@
 {% comment %}
   Example data:
   {
-    block: {
-      entity: {
-        entityBundle: "collapsible_panel"
-        entityRendered: "  <div class="paragraph paragraph--type--collapsible-panel paragraph--view-mode--full">...</div>"
-        entityType: "paragraph"
-      }
+    "entity": {
+        "entityType": "paragraph",
+        "entityBundle": "collapsible_panel",
+        "fieldCollapsiblePanelMulti": false,
+        "fieldCollapsiblePanelExpand": false,
+        "fieldCollapsiblePanelBordered": false,
+        "fieldVaParagraphs": [
+            {
+                "entity": {
+                    "entityId": "391",
+                    "entityBundle": "collapsible_panel_item"
```

Comment:
`entityId` and `entityBundle` are used in combination to make sure the html attribute `id` is unique on the page